.github/workflows/ruby.yml

```diff
@@ -45,3 +45,20 @@ jobs:
 
       - name: RSpec tests
         run: bundle exec rspec --color
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out files
+        uses: actions/checkout@v2
```

Comment:
I thought you were moving to `actions/checkout@v3` (here and above in `test`)?

Reply:
Yes, thanks.